src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactory.java

```diff
@@ -333,10 +351,21 @@ protected void parseInfo(Variant variant, String fileId, String studyId, String
                         break;
                     case "AF":
                         // TODO For now, only one alternate is supported
+                        if(splits[1].eqals("0"))
+                        {
+                        	throw new NotAVariantException;
+                        }
+                        else
+                        {
                         String[] frequencies = splits[1].split(",");
                         file.addAttribute(splits[0], frequencies[numAllele]);
+                        }
                         break;
-//                    case "AN":
+                    case "AN":
+                    		 if(splits[1].eqals("0"))
+                        	{
+                        		throw new NotAVariantException;
```

Comment:
`throw new NotAVariantException();` you have to call the constructor (with parenthesis) to instantiate the class.